apigentools/commands/push.py

```diff
@@ -72,6 +72,14 @@ def run(self):
                     if self.args.skip_if_no_changes and self.git_status_empty():
                         log.info("Only .apigentools file changed for language {}, skipping".format(lang_name))
                         continue
+
+                    # Update git config for this repository to use the provided author's email/name
+                    # If not specified, use the setup from the system/global
+                    if self.args.git_email:
+                        run_command(['git', 'config', 'user.email', self.args.git_email])
```

Comment:
Could you also add `dry_run=self.args.dry_run` to both of these commands?

Reply:
So it looks like the git config command doesn't actually support a `dry-run` option. I got some weird behaviors/things in my git config while testing. So instead I put it as part of the condition for running the command.

Reply:
misunderstood. Didn't see that `run_command` had a dry_run option 😄 Updated